src/test/java/org/cactoos/iterator/HeadOfTest.java

```diff
@@ -60,11 +60,13 @@ public void headIterator() throws Exception {
     public void returnsIntactIterator() throws Exception {
         MatcherAssert.assertThat(
             new LengthOf(
-                new HeadOf<>(
-                    3,
-                    new IterableOf<>(
-                        "one", "two"
-                    ).iterator()
+                new IterableOf<>(
+                    new HeadOf<>(
```

Comment:
@verkhovin @llorllale does it make sense to keep all the classes in the `iterator` package if one can create an `Iterable` from an `Iterator` (and then one can use `HeadOf` and co from the package `iterable` instead of those from the iterator package).